product_print_category_food_report/models/product_product.py

```diff
@@ -221,7 +221,7 @@ def _compute_pricetag_per_unit_quantity_text(self):
                 product.pricetag_per_unit_quantity_text = _("Net Volume")
                 if product.volume > 1:
                     product.pricetag_per_unit_quantity_value = (
-                        _("%.3f L") % product.volume
+                        _("%.2f L") % product.volume
```

Comment:
@legalsylvain la précision type 5.000 L me semble trop précise pour l'étiquette. what do you think ?

Reply:
oui.